packages/core/package.nls.json

```diff
@@ -111,6 +111,7 @@
     "AWS.command.amazonq.fixCode": "Fix",
     "AWS.command.amazonq.optimizeCode": "Optimize",
     "AWS.command.amazonq.sendToPrompt": "Send to prompt",
+    "AWS.command.amazonq.testCode": "Test (Beta)",
```

Comment:
Should this be something like `Generate Tests (Beta)` ? When I right click and see "Test" for some reason I think its going to automagically run my tests for me

Reply:
Yes. I've confirmed with product that "Generate tests (Beta)" makes sense.